js/tweek-local-cache/src/tweek-repository.ts

```diff
@@ -38,18 +38,18 @@ export default class TweekRepository {
   private _client: ITweekClient;
   private _context: Context = {};
   private _getPolicy: GetPolicy;
-  private _refreshInterval: number;
+  private _refreshDelay: number;
   private _isDirty = false;
 
   private _refreshInProgress = false;
   private _refreshPromise: Promise<void>;
   private _nextRefreshPromise: Promise<void>;
 
-  constructor({ client, getPolicy, refreshInterval = 30 }: TweekRepositoryConfig) {
+  constructor({ client, getPolicy, refreshInterval = 30, refreshDelay }: TweekRepositoryConfig) {
```

Comment:
is there a way to mark it as deprecated?
maybe add a `console.warn` when using `refreshInterval`, so that if you update the package it will still work but you will know that you should change it

Reply:
added